canonical-web-and-design/snapcraft.io
# Vanillarise forms

Fixes https://github.com/canonical-web-and-design/snapcraft.io/issues/2177

## Done
- Updated both listing and settings forms to use `p-form p-form--stacked`

## QA
- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/snap_name/listing
- Make sure things look and work as expected (in relation to field spacing &  help text / error text spacing)
- Try and raise some errors on fields
- Visit http://0.0.0.0:8004/snap_name/settings
- Make sure things look and work as expected (as for the listing page)
